imagesgallery/static/html/index.html

```diff
@@ -1,14 +1 @@
-<!doctype html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>React App</title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <script defer="defer" src="bundle.js"></script>
-  </head>
-  <body>
-    <div id="images-gallery-app-root"></div>
-  </body>
-</html>
+<!doctype html><html><head><meta charset="utf-8"/><meta http-equiv="X-UA-Compatible" content="IE=edge"/><title>React App</title><meta name="description" content=""/><meta name="viewport" content="width=device-width,initial-scale=1"/><script defer="defer" src="bundle.js"></script></head><body><div id="images-gallery-app-root"></div></body></html>
```

Comment:
why are we changing the format here?

Reply:
Each time that we make a bundle Webpack generates this file like that